webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -87,8 +72,10 @@ export const ReactionStream = () => {
                     }}
                 />
                 <span>
-                    <Bold>{participants}</Bold>
-                    <span>{untranslatable(' ')}{formatMessage({defaultMessage: 'raised a hand'})}</span>
+                    {formatMessage({defaultMessage: '{users} raised a hand'}, {
+                        count: handsup.length,
+                        users: <Bold>{formatList(handsup.map(getName), {type: 'conjunction', style: 'short'})}</Bold>,
```

Comment:
Is there an easy way to maintain the `x, y, and z` and `x, y, and 3 others` behavior using formatList? From my investigation I couldn't find it (but would be really happy if there was a way). I'm doing this in the DM ringing notification, which is much more complicated:

```
        const additionalNum = otherMembers.length - 2;
        let type: 'conjunction' | 'unit' = 'conjunction';
        if (additionalNum > 0) {
            otherMembers.splice(2);
            type = 'unit';
        }

        message = (
            <FormattedMessage
                defaultMessage={'<b>{hostName}</b> is inviting you to a call with <b>{others}' +
                    '{postscript, select, ' +
                    'true { and {additionalNum, plural, one {1 other} other {{additionalNum} others}}} ' +
                    'other {}' +
                    '}</b>'}
                values={{
                    b: (text: string) => <b>{text}</b>,
                    hostName,
                    others: formatList(otherMembers.map((u) => displayUsername(u, teammateNameDisplay)), {type}),
                    postscript: Boolean(additionalNum > 0),
                    additionalNum,
                }}
            />
        );
```

Reply:
@cpoile apologies for not pushing it yesterday, but this is what I came up with: https://github.com/mattermost/mattermost-plugin-calls/pull/443/files#diff-2fbfe9bc31acb968fab6f96641a9664c8b5f2f444d0679e7bc558086e04f1486R62-R85

It treats the overflow part as a condensing/collapsing effect as a pre-step, where formatList then formats `N others` as a list item itself. 

I'm not 100% sure of how these semantics play out in all languages, but for 9/10+ I think it should work. I tried going down the same path as you for a time —a single formatMessage that handled the `, & N others` part within the same formatMessage, but it was really complicated for the reasons you probably experienced, too.

Reply:
> `x, y, and z` +1 -> `x, y, and 2 others`

[`8698031` (#443)](https://github.com/mattermost/mattermost-plugin-calls/pull/443/commits/8698031790f8f1722f3e1f482fc1d9fb3a5ca94d) adds the dynamic max part

Reply:
I also added a limit to the channel link tooltip (current draft: `10`) as that was previously unbounded